pkg/gui/information_panel.go

```diff
@@ -39,11 +40,28 @@ func (gui *Gui) handleInfoClick() error {
 		return activeMode.Reset()
 	}
 
+	var title, url string
+
 	// if we're not in an active mode we show the donate button
 	if cx <= runewidth.StringWidth(gui.c.Tr.Donate) {
-		return gui.os.OpenLink(constants.Links.Donate)
+		url = constants.Links.Donate
+		title = gui.c.Tr.Donate
 	} else if cx <= runewidth.StringWidth(gui.c.Tr.Donate)+1+runewidth.StringWidth(gui.c.Tr.AskQuestion) {
-		return gui.os.OpenLink(constants.Links.Discussions)
+		url = constants.Links.Discussions
+		title = gui.c.Tr.AskQuestion
+	}
+	err := gui.os.OpenLink(url)
+	if err != nil {
+		// Opening the link via the OS failed for some reason. (For example, this
+		// can happen if the `os.openLink` config key references a command that
+		// doesn't exist, or that errors when called.)
+		//
+		// In that case, rather than crash the app, fall back to simply showing a
+		// dialog asking the user to visit the URL.
+		placeholders := map[string]string{"url": url}
+		message := utils.ResolvePlaceholderString(gui.c.Tr.PleaseGoToURL, placeholders)
+		return gui.PopupHandler.Alert(title, message)
```

Comment:
```suggestion
		return gui.c.Alert(title, message)
```
This is a minor thing relating to the fact that I'm yet to standardise on everything going through this `common` field